README.md

```diff
@@ -1,2 +1,155 @@
+</p>
+
+3. **Stepper motor:**
+<p align="justify">
+A 45kg-cm torque stepper motor is used for the horizontal rotation of the mast cam. The shaft of the stepper motor is attached to a threaded rod with a shaft coupler which transfers the rotation to the top part of the mast.
+</p>
+
+## Research part
+<p align="justify">
+We did an extensive research on the previously sent Mars Rovers and extracted details about the different components of the rover. We looked upon the structure and functioning of the parts and decided accordingly what to put in our rover. Although it was not practical to use all senosors and parts used by high end mars missions, we added whatever was possible in our reach to aid the research work of the rover.  
+</p>
+
+**This is the link of our initial research doc:** https://docs.google.com/document/d/1-jrgd7fFMnukzN-lmV8PeAjKU0p7QLTJImEPwY9DrF8/edit
+
+## Applications
+1. The rover can be used to research the areas with highly uneven terrains and surfaces with ease due to its Rocker bogie suspension system.
+2. It is equipped with a robotic arm and drill which enables sample data collection and storage.
+3. The rover consists of a camera to take 3D images and high definition videos for mapping the terrain as well as analyzing it scientifically.
+4. It can also be used to analyze the collected samples using the various sensors present on the rover
+
+## Limitations
+1. Speed of the rover is very low even on the flat terrains.
+2. Battery life of the rover is short due to less charging capacity.
+
+## Future Improvement
+1. Better materials could be used instead of 3D printed plastic.  
+2. Simulating the model in Gazebo and ROS.  
+
+
+## Team members (with GitHub IDs)
+1. Avinash Singh- <a href="https://github.com/matbot1">matbot1</a>
+2. Harikrishnan PB- <a href="https://github.com/MurkeyCube">MurkeyCube</a>
+3. Pratyush Ojha- <a href="https://github.com/Pratyush-932">Pratyush-932</a>
+4. Sambit Prabhu- <a href="https://github.com/SambitPrabhu">SambitPrabhu</a>
+
+## Mentors:
+1. Vansh Goyal- <a href="https://github.com/vanshgoyal">vanshgoyal</a>
+
+## Reference links
+**Youtube-**  
+How I build mars rover replica- https://www.youtube.com/watch?v=NOZZMsMAGh0&t=142s  
+[Mars Rover 2020 Perseverance] Body Assembly- https://www.youtube.com/watch?v=McydwwR8xu4  
+JPL Mars Science Laboratory The Curiosity Rover- https://www.youtube.com/watch?v=007SnaUxi40  
+
+
+**Research links-**
+https://www.hindawi.com/journals/ijae/2016/5181097/  
+https://mars.nasa.gov/mars2020/spacecraft/rover/  
+https://llis.nasa.gov/lesson/11501  
+http://robotics.estec.esa.int/ASTRA/Astra2004/Papers/astra2004_D-05.pdf  
+https://teamares.sserd.org/project_files/Ares_MCEV_Report.pdf  
```

Comment:
Hyperlink the Github IDs and References properly. No need to link the URLs again. Refer this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

Reply:
> Hyperlink the Github IDs and References properly. No need to link the URLs again. Refer this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

Changes made sir.